IBERTpy/latex/06-08-22_CM203_eyescan_summary.out

```diff
@@ -0,0 +1,20 @@
+\BOOKMARK [0][-]{chapter.1}{Links Between xcvu13p\1370 to xcvu13p\1371 Overview \(1 of 2\)}{}% 1
```

Comment:
Please delete all these latex files (aux, out, etc) from the repo; they are recreated on every compile and aren’t needed. Probably makes sense to add them to the .gitignore.